app/models/cdo/manage/cdo.js

```diff
@@ -34,22 +34,75 @@ function ViewModel (details, cdo, backNav, continueLink) {
     }
   ]
 
-  const modelDetails = mapManageCdoDetails(details, cdo)
+  const modelDetails = mapManageCdoDetails(tasklist, cdo)
+
+  /**
+   * @type {GovukSummaryList[]}
+   */
+  const summaries = [
+    {
+      classes: 'govuk-!-font-size-16',
+      rows: [
+        {
+          key: {
+            text: 'Dog name',
+            classes: 'govuk-!-width-one-half'
+          },
+          value: {
+            text: modelDetails.summary.dogName
+          }
+        },
+        {
+          key: {
+            text: 'Owner name',
+            classes: 'govuk-!-width-one-half'
+          },
+          value: { text: modelDetails.summary.ownerName }
+        }
+      ]
+    },
+    {
+      classes: 'govuk-!-font-size-16',
+      rows: [
+        {
+          key: {
+            text: 'Microchip number',
+            classes: 'govuk-!-width-one-half'
+          },
+          value: {
+            html: [modelDetails.summary.microchipNumber ?? '<span class="defra-secondary-text">Not entered</span>', modelDetails.summary.microchipNumber2].join('<br>'),
```

Comment:
Microchip Number 2 won;t be necessarily in the span styling of 'defra-secondary-text' without your .join() here